_includes/sqlstyle.guide.md

```diff
@@ -277,27 +291,30 @@ FROM my_tmp_table
 
 #### Sub-queries
 
-In PostgreSQL you should probably be doing subqueries with `WITH` clauses.
+In PostgreSQL you should be doing subqueries with `WITH` clauses and avoiding the use of inline
+subqueries.
```

Comment:
I think it's more readable but people can disagree, or we can switch it back to "probably".  [your recent PR|https://github.com/lumoslabs/lumos_data_warehouse/pull/3627/files] provides a good case study for people confused about what they are voting on